### DISCUSSION TITLE
723

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#907] [Comments]
# 723

어제 저녁 우리 회사의 2번째 서비스인 랠릿의 팀원분들과 이야기를 나누었다. 신규 서비스가 런칭하고, 약 1년간 운영을 해왔다. 그리고 계속해서 신규 기능을 넣고 있지만 지표가 내맘같이 오르지 않고 있다. 이럴때 보통 해당 제품의 개발자들은 매일 매일 서비스 지표를 확인하고 일회일비 한다. 그래서 그 부분에 대해 내 생각을 팀원분들께 전달해서 슬랙으로 전달드렸다. 아래 내용은 그 슬랙의 내용 중 민감한 부분을 제외한 전문이다. 최근에 주식 차트 보듯이 매일 매일 서비스 지표를 확인하고 왜 안오르냐고 답답해 하는 모습이 눈에 띄어서 이야기를 드렸어요. 아마 랠릿의 담당자분들은 이런 것에 목말라하실것 같아서 제 이야기를 좀 하고 싶었습니다. 랠릿 이라는 서비스는 런칭한지 이제야 1년 반, 제대로 목적 조직이..

https://jojoldu.tistory.com/723

**Comment by sangwoo-sean** [comment]
일회일비 -> 일희일비 오타났네요
이렇게 건강한 소통을 하는 리드와 일하는 팀원분들이 참 부럽군요. 장거리 마라톤처럼 버텨나가야 한다.. 인상깊게 읽었습니다.

**Reply by jojoldu** [reply]
아고 오타제보 감사합니다 :) 수정해두었습니다!
